com.google.Chrome.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: com.google.Chrome
+desktop-file-name-prefix: '(Testing) '
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: chrome
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--require-version=1.8.1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.chrome.*'
+  - '--filesystem=host-etc'
+  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-videos'
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        branch: spawn-strategy
+        commit: 0c58167d2feb1e8065eafa3df06e7f63043a8a15
+      # - type: dir
+      #   path: ../../code/zypak
+
+  - name: chrome
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 chrome.sh /app/bin/chrome'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 stub_sandbox.sh /app/bin/stub_sandbox'
+      - 'install -Dm 644 -t /app/share/flatpak-chrome flatpak_policy.json'
+      - 'install -Dm 644 -t /app/share/flatpak-chrome first_run.html'
+      - 'install -Dm 644 -t /app/share/flatpak-chrome mimic_warning.txt'
+      - 'install -Dm 644 -t /app/share/applications com.google.Chrome.desktop'
+      - 'install -Dm 644 -t /app/share/metainfo com.google.Chrome.metainfo.xml'
+      - |
+          for icon in 16 24 32 48 64 128 256; do
+            install -Dm 644 com.google.Chrome-$icon.png /app/share/icons/hicolor/${icon}x${icon}/apps/com.google.Chrome.png
+          done
+
+      # For extra-data to be able to extract the .deb
+      - 'install -Dm 755 /{usr,app}/bin/ar'
+      - 'cp /usr/lib/$(gcc --print-multiarch)/libbfd-*.so /app/lib'
+    sources:
+      - type: extra-data
```

Comment:
This needs the flathub-data-checker thing

Reply:
Per the message in the top comment:
> Please don't wire this to extra-data-checker for now due to merge annoyances, I'll be updating it manually while in the beta state.

I'm OK with merging this to beta without it but *not* pushing to stable without it as web browsers as the largest opportunity for most desktop Linux users to evaluate untrusted content (with the issues that come along with that).

Reply:
Yeah I *definitely* wouldn't want this on stable without the external data checker set up yet.